tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -9,6 +9,7 @@ import { LinksEnvironment } from '../../../environment'
 import { config } from '../../../../config'
 import { buildXpathLiteral } from '../../../utils/locator'
 
+const downloadButton = '//button[contains(@id, "preview-download")]'
```

Comment:
there can be multiple download buttons
```suggestion
const downloadPreviewButton = '//button[contains(@id, "preview-download")]'
```